legacy/jacorb/src/main/java/org/jboss/as/jacorb/MigrateOperation.java

```diff
@@ -49,13 +59,21 @@
 
 public class MigrateOperation implements OperationStepHandler {
 
-    private static final OperationStepHandler INSTANCE = new MigrateOperation();
+    public static final String MIGRATE = "migrate";
+    public static final String DESCRIBE_MIGRATION = "describe-migration";
+    public static final String MIGRATION_OPERATIONS = "migration-operations";
+    public static final String MIGRATION_ERROR = "migration-error";
+
 
     static void registerOperation(final ManagementResourceRegistration registry, final ResourceDescriptionResolver resourceDescriptionResolver) {
-        registry.registerOperationHandler(new SimpleOperationDefinitionBuilder(JacORBSubsystemConstants.MIGRATE, resourceDescriptionResolver)
+        registry.registerOperationHandler(new SimpleOperationDefinitionBuilder(MIGRATE, resourceDescriptionResolver)
```

Comment:
I think these need a description of the return type (see my recent web migration commit for an example)